tests/phpunit/tests/post/meta.php

```diff
@@ -227,21 +225,21 @@ function test_update_meta() {
 	 * @see https://core.trac.wordpress.org/ticket/12860
 	 */
 	function test_funky_post_meta() {
-		$classy = new StdClass();
-		$classy->ID = 1;
-		$classy->stringy = "I love slashes\\\\";
-		$funky_meta[] = $classy;
+		$classy          = new StdClass();
+		$classy->ID      = 1;
+		$classy->stringy = 'I love slashes\\\\';
+		$funky_meta[]    = $classy;
 
-		$classy = new StdClass();
-		$classy->ID = 2;
-		$classy->stringy = "I love slashes\\\\ more";
-		$funky_meta[] = $classy;
+		$classy          = new StdClass();
+		$classy->ID      = 2;
+		$classy->stringy = 'I love slashes\\\\ more';
```

Comment:
Not an equivalent value.

Reply:
We are now matching upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/post/meta.php

Reply:
Still, changing the quotes gives a different result.

Reply:
I think I was misreading it. My mistake.